generate.sh

```diff
@@ -108,7 +119,7 @@ CHECKOUT_COMMIT=$LOOP_COMMIT
 COMPONENT=loop
 COMMAND=loop
 DAEMON=loopd
-PROTO_SRC_DIR="swapserverrpc"
+PROTO_SRC_DIR=""
 EXCLUDE_PROTOS="server.proto -not -name common.proto"
```

Comment:
```suggestion
PROTO_SRC_DIR=""
EXCLUDE_PROTOS="./swapserverrpc/*"
```

Exclude the `swapserverrpc` dir when finding proto files to pass to `protoc`.